### ISSUE TITLE
Fail when using version 1.9 on Macbook M1

### ISSUE THREAD

**Issue by lordnox** [julianhille/MuhammaraJS#102]
This should probably not compile 'neon' into muhammara / libpng on my machine (arm64 / m1 chip)

```
dyld: lazy symbol binding failed: Symbol not found: _png_init_filter_functions_neon
  Referenced from: /Users/nox/projects/lft/LifeTime-Desktop/packages/lft-frontend/node_modules/muhammara/binding/muhammara.node
  Expected in: flat namespace
dyld: Symbol not found: _png_init_filter_functions_neon
  Referenced from: /Users/nox/projects/lft/LifeTime-Desktop/packages/lft-frontend/node_modules/muhammara/binding/muhammara.node
  Expected in: flat namespace
/Users/nox/projects/lft/LifeTime-Desktop/packages/lft-frontend/node_modules/electron/dist/Electron.app/Contents/MacOS/Electron exited with signal SIGABRT
```

**Comment by julianhille** [comment]
Another option would be to add the arm c and header files.
I requested to be a macos11 actions tester as I don't have any M1 laying around to test and fix.

**Comment by julianhille** [comment]
Fixed by #104
